Source/WebCore/loader/LinkLoader.cpp

```diff
@@ -112,6 +112,10 @@ void LinkLoader::loadLinksFromHeader(const String& headerValue, const URL& baseU
             continue;
 
         LinkLoadParameters params { relAttribute, url, header.as(), header.media(), header.mimeType(), header.crossOrigin(), header.imageSrcSet(), header.imageSizes(), header.nonce(), ReferrerPolicy::EmptyString };
+        
+        if (!header.referrerPolicy().isEmpty())
+            params.referrerPolicy = *parseReferrerPolicy(header.referrerPolicy(), ReferrerPolicySource::ReferrerPolicyAttribute);
```

Comment:
This is not quite correct. We can’t just unconditionally dereference the result of `parseReferrerPolicy`, because that will just crash at runtime. We need to figure out what to do when it does not parse. Could be as simple as using `value_or` to give it a default value.

Reply:
Probably we want to keep the default value (empty string in that case).
it seems worth adding a test for bad values.